.coafile

```diff
@@ -44,6 +44,6 @@ enabled = False
 bears = LineCountBear
 max_lines_per_file = 1000
 
-[all.PythonPackageInit]
```

Comment:
why is this being removed?

my understanding is that those `__init__.py` files are needed for `setuptools` to correctly find/install the bears.